std/file.d

```diff
@@ -862,7 +862,7 @@ unittest
 version(StdDdoc) void getTimesWin(R)(R name,
                                   out SysTime fileCreationTime,
                                   out SysTime fileAccessTime,
-                                  out SysTime fileModificationTime)
+                                  out SysTime fileModificationTime) @safe
```

Comment:
No, not kidding. I don't regularly do exhaustive searches for documentation versions. I've seen non-template functions that have no body for ddoc, but not templates. It seems like an incorrect solution (obviously given the error here). How can any inference happen?

It seems the version(StdDdoc) trick leaves quite a bit to be desired. Perhaps it needs revisiting. If I were designing it today, I'd want to have a way to document the various compiler versions simultaneously without needing to add special documentation versions of the functions. Otherwise, you may get the WRONG inference!